README.md

```diff
@@ -92,6 +92,6 @@ In brief:
 [github-pages]: https://help.github.com/articles/creating-project-pages-manually/
 [issues]: https://github.com/swcarpentry/lesson-example/issues
 [rendered]: https://swcarpentry.github.io/lesson-example/
-[setup]: ./setup.md
+[setup]: .http://swcarpentry.github.io/lesson-example/setup/
```

Comment:
The dot before `http:` is going to break the link! And I think that GitHub now supports HTTPS so please replace `http` with `https`. Thanks for you contribution to improve our documentation.